typed_python/PyInstance.cpp

```diff
@@ -587,13 +597,15 @@ PyTypeObject* PyInstance::typeObj(Type* inType) {
 // static
 PySequenceMethods* PyInstance::sequenceMethodsFor(Type* t) {
     if (    t->getTypeCategory() == Type::TypeCategory::catAlternative ||
-            t->getTypeCategory() == Type::TypeCategory::catConcreteAlternative) {
+            t->getTypeCategory() == Type::TypeCategory::catConcreteAlternative ||
+            t->getTypeCategory() == Type::TypeCategory::catClass
+            ) {
         PySequenceMethods* res =
             new PySequenceMethods {0,0,0,0,0,0,0,0};
             res->sq_contains = (objobjproc)PyInstance::sq_contains;
             res->sq_length = (lenfunc)PyInstance::mp_and_sq_length;
-            res->sq_item = (ssizeargfunc)PyInstance::sq_item;
-            res->sq_ass_item = (ssizeobjargproc)PyInstance::sq_ass_item;
+            //res->sq_item = (ssizeargfunc)PyInstance::sq_item;
```

Comment:
i'm curious why do we not have these any more? are they superseded by other methods or something?

Reply:
[ ] could be the SequenceMethod sq_item/sq_ass_item or it could be the MappingMethod mp_subscript/mp_ass_subscript.  The SequenceMethod methods require an integer index, while the MappingMethod methods are less restrictive.  Since I don't know if the Alternative or Class is emulating a Sequence or a Map, I went with the less restrictive signature.